components/private/Icons/icons/ic_add/src/MaterialIcons+ic_add.h

```diff
@@ -0,0 +1,38 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialIcons.h"
+
+// This file was automatically generated by running ./scripts/sync_icons.sh
+// Do not modify directly.
+
+@interface MDCIcons (ic_add)
```

Comment:
This add a symbol to MDC that is not actually used by our components.  I would expect image this to be a part of the example, not pulled from our private icons component.
One of the goals of the examples is to allow people to easily utilize the code in their own projects.  Relying on a private API makes this a bit trickier.

Reply:
Fair enough, but since I just wanted to demonstrate the issue, I didn't think of the cleanest solution.